codegen/java-gen/src/test/scala/com/lightbend/akkasls/codegen/java/SourceGeneratorSuite.scala

```diff
@@ -179,56 +179,52 @@ class SourceGeneratorSuite extends munit.FunSuite {
-        |                .registerEventSourcedEntity(
-        |                    MyEntity3.class,
-        |                    ServiceOuterClass3.getDescriptor().findServiceByName("MyService3"),
-        |                    EntityOuterClass3.getDescriptor(),
-        |                    ExternalDomain.getDescriptor()
-        |                )
-        |                .registerView(
-        |                    MyService4.class,
-        |                    ServiceOuterClass4.getDescriptor().findServiceByName("MyService4"),
-        |                    "my-view-id4",
-        |                    EntityOuterClass4.getDescriptor()
-        |                );
-        |    }
-        |}""".stripMargin
+      """|/* This code is managed by Akka Serverless tooling.
+         | * It will be re-generated to reflect any changes to your protobuf definitions.
+         | * DO NOT EDIT
+         | */
+         |
+         |package com.example.service;
+         |
+         |import com.akkaserverless.javasdk.AkkaServerless;
+         |import com.example.service.persistence.EntityOuterClass1;
+         |import com.example.service.persistence.EntityOuterClass2;
+         |import com.example.service.persistence.EntityOuterClass3;
+         |import com.example.service.persistence.MyEntity1;
+         |import com.example.service.persistence.MyEntity3;
+         |import com.example.service.persistence.MyValueEntity2;
+         |import com.example.service.persistence.MyValueEntity2Provider;
+         |import com.example.service.something.MyService3;
+         |import com.example.service.something.ServiceOuterClass3;
+         |import com.example.service.view.MyService4;
+         |import com.example.service.view.ServiceOuterClass4;
+         |import com.external.ExternalDomain;
+         |
+         |public final class SomeMainComponentRegistrations {
+         |
+         |  public static AkkaServerless withGeneratedComponentsAdded(AkkaServerless akkaServerless) {
+         |    return akkaServerless
+         |      .registerEventSourcedEntity(
+         |          MyEntity1.class,
+         |          ServiceOuterClass1.getDescriptor().findServiceByName("MyService1"),
+         |          EntityOuterClass1.getDescriptor(),
+         |          ExternalDomain.getDescriptor()
+         |      )
+         |      .register(new MyValueEntity2Provider(MyValueEntity2::new))
+         |      .registerEventSourcedEntity(
+         |          MyEntity3.class,
+         |          ServiceOuterClass3.getDescriptor().findServiceByName("MyService3"),
+         |          EntityOuterClass3.getDescriptor(),
+         |          ExternalDomain.getDescriptor()
+         |      )
+         |      .registerView(
+         |          MyService4.class,
+         |          ServiceOuterClass4.getDescriptor().findServiceByName("MyService4"),
+         |          "my-view-id4",
+         |          EntityOuterClass4.getDescriptor()
+         |      );
```

Comment:
btw, we were not testing the `registerAction`

Didn't add it, but we should do it soon.